components/multiselect/multiselect.jsx

```diff
@@ -150,6 +155,17 @@ export default class MultiSelect extends React.Component {
             );
         }
 
+        const memberCount = (
+            <FormattedMessage
+                id='multiselect.numRemaining'
```

Comment:
This needs its own internationalization ID like `multiselect.numMembers`, and then that needs to be added to `i18n/en.json` with the matching message like the other localized strings